bin/bootstrap/ubuntu/14.04.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-
+set -x
```

Comment:
Also `set -e` I think you can combine them with `set -ex` `-e` will cause the script to fail on the first error, which is probably what you want.